docs/articles_en/learn-openvino/llm_inference_guide/llm-inference-hf.rst

```diff
@@ -235,17 +235,25 @@ includes **Dynamic quantization** of activations of 4/8-bit quantized MatMuls an
   insignificant deviation in generation accuracy.  Quantization is performed in a group-wise
   manner, with configurable group size. It means that values in a group share quantization
   parameters. Larger group sizes lead to faster inference but lower accuracy. Recommended
-  group size values are ``32``, ``64``, or ``128``. To enable Dynamic quantization, use
-  the corresponding inference property as follows:
+  group size values are ``32``, ``64``, or ``128``. Dynamic quantization is enabled **by default**
+  on the CPU device. To disable dynamic quantization, change execution mode from the ``PERFORMANCE mode``
```

Comment:
You can also set DYNAMIC_QUANTIZATION_GROUP_SIZE to 0

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/26004/commits/7bb2e8bd4943465c4cba6614fa2b104bbe14e8d4

Reply:
I would recommend to document DYNAMIC_QUANTIZATION_GROUP_SIZE=0 is primary option to disable DQ optimization. 
ACCURACY execution mode is also an option, but if affects inference precision as well, which may highly affect inference performance (esp on Xeon CPUs and Intel GPUs)